package.json

```diff
@@ -1,46 +1,18 @@
 {
-  "name": "doc-ops",
+  "name": "documentation",
   "version": "0.0.0",
-  "private": true,
+  "private": "true",
   "scripts": {
-    "docusaurus": "docusaurus",
-    "start": "docusaurus start",
-    "build": "docusaurus build",
-    "swizzle": "docusaurus swizzle",
-    "deploy": "docusaurus deploy",
-    "clear": "docusaurus clear",
-    "serve": "docusaurus serve",
-    "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids"
+    "start": "iota-wiki-cli start",
+    "clean": "iota-wiki-cli clean",
+    "setup": "iota-wiki-cli setup",
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.6f366f4b4",
-    "@docusaurus/preset-classic": "^2.0.0-beta.6f366f4b4",
-    "@mdx-js/react": "^1.6.21",
-    "@svgr/webpack": "^5.5.0",
-    "clsx": "^1.1.1",
-    "file-loader": "^6.2.0",
-    "prism-react-renderer": "^1.2.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "url-loader": "^4.1.1"
-  },
-  "resolutions": {
-    "trim": ">=0.0.3",
-    "glob-parent": "^5.1.2",
-    "browserslist": "4.16.5",
-    "css-what": ">=5.0.1"
+  "license": "UNLICENSED",
+  "engines": {
+    "node": ">=14.14.0"
   },
-  "browserslist": {
-    "production": [
-      ">0.5%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "dependencies": {
+    "iota-wiki-cli": "git+https://github.com/iota-community/iota-wiki-cli.git#v1.7.0"
```

Comment:
Could also use the npm package but as we move everything in the near future anyway we need to update everything again then anyways 😅